src/framework/preferences/js/Enactors.js

```diff
@@ -427,4 +427,244 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
+
+    fluid.prefs.enactor.muteAudio.applyMuteAudio = function (value) {
+        $(document).ready(function () {
+            var audios = $("audio, video");
+            if (value) {
+                audios.each(function () {
+                    if (!$(this).prop("muted")) {
+                        $(this).prop("muted", true);
+                        mutedComponents.push(this);
+                    }
+                });
+            }
+            else {
+                audios.each(function () {
+                    var index = mutedComponents.indexOf(this);
+                    if (index > -1) {
+                        $(this).prop("muted", false);
+                        mutedComponents.splice(index, 1);
+                    }
+                });
+            }
+        });
+    };
+
+    /*******************************************************************************
+     * blueColorFilter
+     *
+     * Sets the intensity of the blue color of the page to the multiple provided.
+     *******************************************************************************/
+
+    // Note that the implementors need to provide the container for this view component
+    fluid.defaults("fluid.prefs.enactor.blueColorFilter", {
+        gradeNames: ["fluid.prefs.enactor", "fluid.viewComponent"],
+        preferenceMap: {
+            "fluid.prefs.blueColorFilter": {
+                "model.value": "default"
+            }
+        },
+        expander: {
+            func: "{that}.addMutationObserver"
+        },
+        invokers: {
+            applyFilter: {
+                funcName: "fluid.prefs.enactor.blueColorFilter.applyFilter",
+                args: ["{that}.model.value"]
+            },
+            addMutationObserver: {
+                funcName: "fluid.prefs.enactor.blueColorFilter.addMutationObserver",
+                args: ["{that}.applyFilter"]
+            }
+        },
+        modelListeners: {
+            value: {
+                listener: "{that}.applyFilter"
+            }
+        }
+    });
+
+    /*Used in fluid.prefs.enactor.blueColorFilter.applyFilter and fluid.prefs.enactor.blueColorFilter.addMutationObserver
+     had to be defined here in order to not be create every time the model is changed */
+    var initialColorsDictionary = [];
+    var noMatchFlag = true;
+    var changedDOMFlag = false;
+    var blueColorFilterValue;
+
+    fluid.prefs.enactor.blueColorFilter.applyFilter = function (times) {
+        // Prevents a bug when the panel is open and the model.value becomes undefined
+        if (times === undefined) {
+            times = blueColorFilterValue;
+        }
+        blueColorFilterValue = times;
+        var elements = $("body *");
+        var index;
+        elements.each(function () {
+            var element = this;
+            // Checks if the element exists in the dictionary with keys the elements and values the colors
+            for (var count in initialColorsDictionary) {
```

Comment:
I tried to replace all the for-s with fluid.each, but in one case it didn't work. Is there a break equivalent for the fluid.each function?